test/unit/mock/cmd/bash-c-type-nginx

```diff
@@ -0,0 +1 @@
+nginx is /usr/sbin/nginx
```

Comment:
Not quite... I'm suggesting that you can mock the `inspec.command` call with a minitest mock so that the InSpec MockLoader stuff never gets called. Then you have full control over the responses returned to the method when you're testing.

You can look at the unit tests for the `host` resource for some examples.

 * set up an `inspec` object: https://github.com/chef/inspec/blob/master/test/unit/resources/host_test.rb#L190
 * stub it for the test: https://github.com/chef/inspec/blob/master/test/unit/resources/host_test.rb#L194-L196
 * stub the "command(blah).exist?" calls: https://github.com/chef/inspec/blob/master/test/unit/resources/host_test.rb#L194-L196